modules/amrfinder_report.nf

```diff
@@ -18,6 +20,8 @@ process AMRFINDER_REPORT {
     amr_report.py \
     --amr_out ${amrfinder_tsv} \
     --mobilome ${mobilome_gff} \
+    --contigs_map ${map_file} \
+    --user_gff ${user_gff}
```

Comment:
user_gff should not be send to the script if it's "NO_FILE".

Like:
```
script:
def user_gff_arg = user_gff.name != "NO_FILE" ? "--user_gff ${user_gff} : ""
"""
arm_report.py ...
--config_map ${map_file} ${user_gff_arg}
"""
```

Reply:
Nice solution :D